pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> If we want to create a new type TerminationNone that would be more accurate
> (and default to that).  However, a user who creates and adds certs almost
> certainly wants termination, not "none"

I'm not sure it makes sense to have a `TLSConfig` object, whose sole purpose is provide TLS termination information with a flag on it that means, "don't use the TLSConfig I provided".